src/test/scala/com/sandinh/paho/akka/BrokerHelper.scala

```diff
@@ -0,0 +1,24 @@
+package com.sandinh.paho.akka
+import sys.process._
+
+trait BrokerHelper {
+  protected def startBroker(logPrefix: String = null, port: Int = 1883): Process = {
+    val brokerFile = {
+      // use /usr/local/sbin/mosquitto (if it exists) before /usr/sbin/mosquitto
+      if (scala.reflect.io.File(scala.reflect.io.Path("/usr/local/sbin/mosquitto")).exists)
```

Comment:
we should just use `java.io.File`:
`new File("/usr/local/sbin/mosquitto").exists()`